seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -1209,9 +1210,6 @@
         "assert($('body').css('font-family') == '\"Open Sans\", sans-serif', 'message: The <code>body</code> tag should have the font of \"Open Sans\".');"
```

Comment:
@dhcodes while you are at it can you update the message as well, to include secondary font. Or add a new test case, whatever you feel like is better?

I have resolved a conflict, so please pull first from your remote branch.